test/resources/examples/index.js

```diff
@@ -156,6 +195,8 @@ module.exports = Object.assign(
     biggieSmallTrade,
     basicRingTrade,
     shortRingBetterTrade,
+    tooSmallBuyAmountTrade,
+    tooSmallSellAmountTrade,
```

Comment:
Do we really want to expose invalid examples? Maybe the generic example file should only contain valid cases? At least they should be in a separate file so that when we test them, we know we have to expect a failure or trivial solution. 

Wouldn't we be able to take a valid case and scale it down by so much (dividing by 1000 or so) that the values become too small?

Reply:
I could make a separate PR to address this. Since these examples are used only once I might even go as far as saying that they don't need to be constructed as a JS object in the first place. I could put these examples directly into their relevant unit tests. How does that sound?